LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/JsonHandlers/LnsStationConfiguration.cs

```diff
@@ -157,6 +158,14 @@ select JoinEui.Parse(eui))
                                     WriteRouterConfig(netId, joinEui, region, hwspec, freqRange, drs,
                                                       sx1301conf, nocca, nodc, nodwell));
 
+        private static readonly IJsonReader<Region> RegionConfigurationConverter =
+            JsonReader.Object(JsonReader.Property("region", from s in JsonReader.String()
+                                                            select Enum.TryParse<LoRaRegionType>(s, out var loraRegionType)
```

Comment:
Should we consider adding a `JsonReader.Enum<T>` that would allow us to reuse this logic?

Reply:
Added #712